content/blog/2020-outreach-partners.md

```diff
@@ -0,0 +1,31 @@
+Title: Bringing PyCon India to a wider audience with your help
+Date: 2020-07-23 13:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-outreach-partners
+
+[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about the conference in their communities and help bring more fresh faces to the event.
+
+<!-- PELICAN_END_SUMMARY -->
+
+We are the largest gathering of Python users and admirers in India who come together once a year to celebrate the language and share our experience and expertise with the community.
+
+Let's make make PyCon India a great conference using the combined effort of our communities. 
+
+## What we expect from you 
+
+We would love for you to spread a word about the conference among your members who are looking to learn something regarding Python. We expect our outreach partners to help us in the following ways: 
+
+* Send emails to your mailing lists and post content on their social media channels on our behalf to make your community members aware of the conference. 
+* Spread the word about our call for proposals for talks, workshops & Dev sprint and help us find speakers, mentors, and reviewers from your community for the conference.
+
+## What you can expect from us
+
+For supporting the conference, our outreach partners can expect following in return from PyCon India:
+
+* We will display your Logos and Names on the conference website. 
+* We will also promote your community on our social media channels as a supporter of the conference.
+
+If you are an organization or a community interested in becoming an outreach partner for PyCon India 2020, please reach out to us at [link]

```

Comment:
@anistark please suggest the [link]

Reply:
@abhishekmishragithub @palnabarun Do we've an email for this? like partnership(s) [at] .... ??

Reply:
@anistark we don't have an email id for it. We can use contact@in.pycon.org  (from last year)

Reply:
Who gets contact emails? @abhishekmishragithub

Reply:
We need to add people to alias, @palnabarun or @sayanchowdhury can help.
It was already set up last year by Vijay (@bravegnu)

Reply:
Till then, let's use a google form or something? Less dependency.

Reply:
+1 more control, transparency, and lower dependency on just one person. 
@anistark maybe we can wait for comments or do provide the form link

Reply:
@anistark @abhishekmishragithub Need the form link folks and finish this PR.